proto/src/scim_v1/server.rs

```diff
@@ -3,8 +3,9 @@ use super::ScimOauth2ClaimMapJoinChar;
 use super::ScimSshPublicKey;
 use crate::attribute::Attribute;
 use crate::internal::UiHint;
+use log::debug;
```

Comment:
Tracing?

Reply:
I don't see tracing in proto, should I add it or should those functions not log

Reply:
Add tracing because that's what we use everywhere.